dev/com.ibm.ws.opentracing/resources/com/ibm/ws/opentracing/resources/Opentracing.nlsprops

```diff
@@ -86,6 +86,14 @@ OPENTRACING_TRACERFACTORY_RETURNED_NULL=CWMOT0006E: Invocation of user supplied
 OPENTRACING_TRACERFACTORY_RETURNED_NULL.explanation=The feature constructs Tracer implementations with a user provided OpentracingTracerFactory. The OpentracingTracerFactory.newInstance method returned null.
 OPENTRACING_TRACERFACTORY_RETURNED_NULL.useraction=Determine why the user supplied OpentracingTracerFactory newInstance method returned null.
 
+OPENTRACING_FILTER_PATTERN_BLANK=CWMOT0007E: The pattern attribute of an opentracing excludeSpans or includeSpans configuration element is blank.
+OPENTRACING_FILTER_PATTERN_BLANK.explanation=A valid URI pattern must be specified.
+OPENTRACING_FILTER_PATTERN_BLANK.useraction=Consult the documentation on valid patterns.
+
+OPENTRACING_FILTER_PATTERN_INVALID=CWMOT0008E: The pattern attribute {0} of an opentracing excludeSpans or includeSpans configuration element is an invalid URI (RFC 2396).
+OPENTRACING_FILTER_PATTERN_INVALID.explanation=A valid URI pattern must be specified.
+OPENTRACING_FILTER_PATTERN_INVALID.useraction=Consult RFC 2396. The most common problem is not properly % URL-encoding invalid characters.
+
```

Comment:
**Overall these messages look good. Since the "user action" directs a user to perform something, I'd recommend moving the information to the "explanation" about the common problem with the % URL-encoding invalid characters to that message instead (see the following).**

+OPENTRACING_FILTER_PATTERN_INVALID.explanation=A valid URI pattern must be specified. The most common problem is not properly % URL-encoding invalid characters.
+OPENTRACING_FILTER_PATTERN_INVALID.useraction=**Consult RFC 2396 and ensure that you have correctly % URL-encoding invalid characters.**

Reply:
Updated in the latest commit.